### PR TITLE
Feature/chat locker 24

### PR DESCRIPTION
**Pull request by rwakida**
Observes Users collection and adds or removes a user based on their location to the associated location channel.  If the channel does not exist, then it creates the channel.  Like the "General" channel, the location channels have no owner.

### CODE REVIEW

File: server/startup/jedis.js [hunk 1]
```diff
@@ -37,8 +44,104 @@ Meteor.startup( function() {
...
+			})
+		}
+
+	})
+}
+
+var addUserToLocationChannel = function(user) {
+	var channel;
+	var chId;
+	if( _.isUndefined(user.profile) || _.isUndefined(user.profile.location) ) {
```
Comment (bbrockman) [inline server/startup/jedis.js:73]: I think we should add the username to the log.

Comment (rwakida) [inline server/startup/jedis.js:73]: Do you mean to the console.log or to a message in the room that the user has joined/left?

Comment (bbrockman) [inline server/startup/jedis.js:73]: Just the console.  The next step would be to email an admin that their user data is incomplete.  That is not our concern.

**Comment by douglasrapp** [comment]
I like it. Code looks good. I played around with the users collection in Mongo and channels are created/modified accordingly in real time.

**Pull request merged**